scripts/checks.sh

```diff
@@ -1,17 +1,16 @@
 #!/bin/bash
 RED='\033[0;31m'
 
-echo ':: Get dependencies ::'
-fvm flutter pub get
+echo ':: Sorting translation files ::'
+
+fvm flutter pub run arb_utils:sort lib/l10n/intl_en.arb; 
 
 echo ':: Check code format ::'
-fvm flutter format --set-exit-if-changed . || { echo -e "${RED}Invalid format" ; exit 1; }
+fvm flutter format --set-exit-if-changed . || { 
+  echo -e "${RED}Invalid format" ; exit 1; 
+}
```

Comment:
is it ok? it's usually do in the same line

Reply:
Yeap, its the same